pkg/controller/handlers/cronjob/cronjob_test.go

```diff
@@ -0,0 +1,72 @@
+package cronjob
+
+import (
+	"testing"
+	"time"
```

Comment:
shall we `import  _ time/tzdata ` , so `time.LoadLocation()`  wouldn't rely on the timezone info from OS system? I encountered a similar issue here https://github.com/obot-platform/tools/pull/626/files